validator/testcases/javascript/predefinedentities.py

```diff
@@ -64,6 +101,12 @@
 ]
 
 
+def is_shared_scope(t, r=None, rn=None):
```

Comment:
It's set to a constant value in a lot of places. The only other place it's given a function is:

```py
    u"top": {"readonly": actions._readonly_top},
```

Though that case just emits a notice.

But that's only two instances, so I guess it shouldn't be a problem to change the argument names after all. I was thinking of `"dangerous"`, where lambdas are used much more often.